x-pack/plugins/observability_solution/observability_logs_explorer/public/state_machines/origin_interpreter/src/notifications.tsx

```diff
@@ -5,21 +5,24 @@
  * 2.0.
  */
 
+import React from 'react';
+
 import { IToasts } from '@kbn/core-notifications-browser';
-import { mountReactNode } from '@kbn/core-mount-utils-browser-internal';
+import { toMountPoint } from '@kbn/react-kibana-mount';
 import { EuiButton, EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
 import { i18n } from '@kbn/i18n';
 import { LOGS_ONBOARDING_FEEDBACK_LINK } from '@kbn/observability-shared-plugin/common';
-import React from 'react';
+import { useKibanaContextForPlugin } from '../../../utils/use_kibana';
 import { FEEDBACK_TOAST_LIFETIME_MS } from './constants';
 
 export const createRequestFeedbackNotifier = (toasts: IToasts) => () => {
+  const startServices = useKibanaContextForPlugin().services;
```

Comment:
This change is currently breaking the UI on the touched feature, as we want a notification to appear when coming from the onboarding flow.

| Current change | Expected |
|---|---|
| <img width="1013" alt="Screenshot 2024-05-27 at 10 42 35" src="https://github.com/elastic/kibana/assets/34506779/43e0f9ed-e765-4e8f-8aa4-dbda5e2f32c3"> | <img width="380" alt="Screenshot 2024-05-27 at 10 50 24" src="https://github.com/elastic/kibana/assets/34506779/7db5834d-e1c3-48a9-a51a-81939faaff11"> |

This happens because the context for `useKibanaContextForPlugin` is not wrapping the components using this helper in its state machine.

One solution would be to pass down the whole services object from the top level [LazyOriginInterpreter](https://github.com/elastic/kibana/blob/main/x-pack/plugins/observability_solution/observability_logs_explorer/public/routes/main/main_route.tsx#L55) and pass it down till this helper instead of relying on this context, which is not initialized yet at this level.